api/docs/release.dox

```diff
@@ -153,6 +153,10 @@ Further non-compatibility-affecting changes include:
  - Added new fields to #dr_os_version_info_t which contain the build number,
    edition, and Windows 10 release identifier.
  - Added the function instr_is_xsave().
+ - Added the AVX-512 registers DR_REG_XMM16 - DR_REG_XMM31, DR_REG_YMM16 - DR_REG_YMM31
```

Comment:
(Repeated comment!!) Use `#` prefix for a link.

Reply:
Done.